2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,52 @@
+#__BY:sirjan13__
+
+def gcd(n,m):
+    if n==0:
+        return m
+    return gcd(m%n,n)
+
+t1,t2=0,0
+
+def egcd(a, b):
+    x,y, u,v = 0,1, 1,0
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpufcxdz6g/2017/JULY/JULY17/EXPTREE/EXPTREE.py
+++ b/tmp/tmpufcxdz6g/2017/JULY/JULY17/EXPTREE/EXPTREE.py
@@ -8,11 +8,11 @@
 t1,t2=0,0
 
 def egcd(a, b):
-    x,y, u,v = 0,1, 1,0
+    x, y, u, v = 0, 1, 1, 0
     while a != 0:
-        q, r = b//a, b%a
+        q, r = b//a, b % a
         m, n = x-u*q, y-v*q
-        b,a, x,y, u,v = a,r, u,v, m,n
+        b, a, x, y, u, v = a, r, u, v, m, n
     gc = b
     return gc, x, y
     
```